python_modules/dagster/dagster/core/asset_defs/decorators.py

```diff
@@ -282,6 +289,7 @@ def multi_asset(
     partition_mappings: Optional[Mapping[str, PartitionMapping]] = None,
     op_tags: Optional[Dict[str, Any]] = None,
     can_subset: bool = False,
+    # TODO - does this need a group name too?
```

Comment:
I think we don't need to add a `group_name` field to `multi_asset` for now. People use `@multi_asset` much more rarely than `@asset`, and this field can be easily added later